capsules/src/driver.rs

```diff
@@ -36,6 +36,7 @@ pub enum NUM {
     Rng                   = 0x40001,
     Crc                   = 0x40002,
     Hmac                  = 0x40003,
+    Ctap                  = 0x40004,
```

Comment:
I don't think "cryptography" is the right category for that. It's more a transport layer than cryptography (the cryptography happens in the userspace). Also, this should be named CtapUsb or CtapHidUsb, as it's possible to use CTAP over other protocols (Bluetooth, NFC), which would be different drivers.

Reply:
I have changed the name to `CtapHid`.

I see what you mean about it not being cryptography, but I still feel that this is the right section. Although it isn't cryptography itself, it is very related to it.

Reply:
Well, I don't feel that this is the right section. Should driver number assignments be reviewed and agreed upon by the Tock core team (e.g. in a weekly call)?

Reply:
On the other hand, I don't think these categories have been of much use, and from Tock 2.0 on we could simply allocate driver numbers linearly instead. This would enable some potential optimizations that have been discussed such as storing the driver number on 16 bits, and the sub-driver number on 16 bits as well, both being packed in a single 32-bit register.

Reply:
Agreed. The driver assignment is pretty unimportant, just as long as it doesn't change. libtock-rs is already using this driver number so I would like to avoid changing it unless we have to.

I also agree with 16-bit numbers as part of Tock 2.0.